charts/incubator/speedtest-tracker/Chart.yaml

```
@@ -42,4 +42,4 @@ sources:
   - https://github.com/truecharts/charts/tree/master/charts/incubator/speedtest-tracker
   - https://ghcr.io/linuxserver/speedtest-tracker
 type: application
-version: 0.0.2
+version: 0.1.0
```

```suggestion
version: 0.0.1
```

This is a patch/fix not a new feature.